SPRCLI/Sources/SPR/UploadMetadata.swift

```diff
@@ -38,7 +38,7 @@ extension SPRPublisher {
         jsonEncoder.outputFormatting = [.prettyPrinted, .sortedKeys]
         let data = try jsonEncoder.encode(metadata)
         let body = ByteStream.data(data)
-        let input = PutObjectInput(body: body, bucket: bucket, contentType: "application/json", key: metadataKey)
+        let input = PutObjectInput(body: body, bucket: bucket, cacheControl: "public, max-age=3600, immutable", contentType: "application/json", key: metadataKey)
```

Comment:
Metadata is cached for one hour.  I plan to relax this to a day (86400 sec) when we get to production.